core/lib/spree/event/subscriber.rb

```diff
@@ -54,7 +54,14 @@ def self.included(base)
       #     end
       #   end
       def event_action(method_name, event_name: nil)
-        mattr_accessor "#{method_name}_handler"
+        mattr_writer "#{method_name}_handler"
```

Comment:
Why don't we need to deprecate the writer as well?

Reply:
We still use the writer internally to keep compatibility, if somebody is still using the reader in their codebase (we don't use the reader anymore). So, if we deprecate the writer, then we'll need to silence the deprecation [when using it](https://github.com/solidusio/solidus/pull/3758/files#diff-d6a7a37486ba2958ea8f0d091d2df618R78). So, deprecating the writer would be impractical at this point.

But I think we don't really need to deprecate the writer as well. The only reason for some developers to be using the writer in their codebase is to set a value to be accessed from the reader. When this happens, they will see the deprecation when the matching reader is called.

Reply:
Makes sense, thanks Andrea.